### PR TITLE
618 added intro

### LINKED ISSUE #618: Documentation chapter 1: Purpose/Intro
## Description
In a few short sentences explain for what purpose is this project created in a non-technical language understandable by everyone.

### PR DESCRIPTION
**Pull request by Zejnilovic**
- Added index.md intro
- Move page version script to its own include file
- Added jekyll-redirect-from plugin
- Redirects to documentation without specifying the version possible. Will redirect you to the latest version
- create docs script also automatically transfers the redirects in front matter

Closes #618

### CODE REVIEW

File: _docs/2.0.0/plugins.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: docs
-title: Built-in Plugins
+title: Plugins
 version: '2.0.0'
 categories:
     - '2.0.0'
+redirect_from: /docs/plugins-built-in
```
Comment (dk1844) [inline _docs/2.0.0/plugins.md:7]:
```suggestion
redirect_from: /docs/plugins
```
Btw. where are these paths coming from anyway?

Comment (Zejnilovic) [reply _docs/2.0.0/plugins.md]: These paths are set in `_config.yml` in the root of `gh-pages` branch. They are defined by the collection `docs`. 
The default/latest path/permalink is something @benedeki asked for so we could put non-changeable links to the latest docs.

Does this make sense?

**Comment by Zejnilovic** [comment]
SonarCloud is giving me mixed vibes here. Seems I failed successfully 

image

**Comment by dk1844** [comment]
> SonarCloud is giving me mixed vibes here. Seems I failed successfully
> 
> image

This happened to me as well one time (first run got cancelled somehow, the second finished fine) and there was no way I could find to convince Sonar to propagate the change :(

File: index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: Enceladus
+---
+
+Enceladus is an open source project under the [ABSA Group Limited][absa] designed for transforming
+any data format into a standardized and strongly type `parquet` format and applying a number of
```
Comment (benedeki) [inline index.md:7]:
```suggestion
any data format into a standardized and strongly typed `parquet` format and applying a number of
```

File: index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+conformance rules on top of said data.
+
+Enceladus laverages [Apache Spark][spark] to read and transform the data. On top of that it also
+uses [Cobrix][gh-cobrix] to be able to read COBOL data, [Spline][gh-spline] to keep track of the
+data lineage, and many other open sourrce tools.
+
+Enceladus is shipped as a UI for managing data sets and 2 spark jobs for transforming said data sets.
```
Comment (benedeki) [inline index.md:14]: What about the API part of Menas?

File: _docs/2.0.0/build-process.md [hunk 4]
```diff
@@ -4,4 +4,5 @@ title: Build Process
 version: '2.0.0'
 categories:
     - '2.0.0'
+redirect_from: /docs/build-process
```
Comment (benedeki) [inline _docs/2.0.0/build-process.md:7]: What does this do?

File: _includes/page_version.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{% if page.version %}
+  {% assign v = page.version %}
```
Comment (benedeki) [inline _includes/page_version.html:2]: Where is the `v` actually used? I don't see it even in `navigation.html` where it was used before.
Also a one letter variable, furthermore global it seems, that's rather obscuring.

**Comment by benedeki** [comment]
> 
> 
> > SonarCloud is giving me mixed vibes here. Seems I failed successfully
> > image
> 
> This happened to me as well one time (first run got cancelled somehow, the second finished fine) and there was no way I could find to convince Sonar to propagate the change :(

You can take hints from https://github.com/AbsaOSS/enceladus/pull/1270#issuecomment-611492720 😉

File: index.md (hunk 3 above)
Comment (Zejnilovic) [reply index.md:14]: Not supported currently and should not be mentioned before the audit.

File: _docs/2.0.0/build-process.md (hunk 4 above)
Comment (Zejnilovic) [reply _docs/2.0.0/build-process.md:7]: It does the redirect you wanted, is it not?

So `URL/docs/build-process` and `URL/docs/LATEST-VERSION/build-process` is the same

File: _includes/page_version.html (hunk 5 above)
Comment (Zejnilovic) [reply _includes/page_version.html:2]: still in navigation

File: _docs/2.0.0/usage-schema.md [hunk 6]
```diff
@@ -4,44 +4,51 @@ title: Usage
...
+    - [pattern](#pattern)
+    - [timezone](#timezone)
+    - [decimal_separator](#decimalseparator)
+    - [grouping_separator](#groupingseparator)
+    - [minus_sign](#minussign)
+    - [allow_infinity](#allowinfinity)
+    - [radix](#radix)
+  - [Parsing](#parsing)
+    - [Parsing timestamps and dates](#parsing-timestamps-and-dates)
+      - [Time Zone support](#time-zone-support)
```
[the remark this answers is not in the document]
Comment (Zejnilovic) [reply _docs/2.0.0/usage-schema.md:43]: This I don't remember doing, I think it was automatic.

**Comment by Zejnilovic** [comment]
> > > SonarCloud is giving me mixed vibes here. Seems I failed successfully
> > 
> > 
> > This happened to me as well one time (first run got cancelled somehow, the second finished fine) and there was no way I could find to convince Sonar to propagate the change :(
> 
> You can take hints from [#1270 (comment)](https://github.com/AbsaOSS/enceladus/pull/1270#issuecomment-611492720) 😉

Sorry, I don't get it.

File: _includes/page_version.html (hunk 5 above)
Comment (Zejnilovic) [reply _includes/page_version.html:2]: actually can be put back

File: _docs/2.0.0/build-process.md (hunk 4 above)
Comment (benedeki) [reply _docs/2.0.0/build-process.md:7]: Ah so it instructs the build to create the redirection from RL/docs/build-process toURL/docs/LATEST-VERSION/build-process? Cool 👍

File: _includes/page_version.html (hunk 5 above)
Comment (benedeki) [reply _includes/page_version.html:2]: > 
> 
> still in navigation

Ah I see. IMO would make better sense then to be defined somewhere and used elsewhere. Also what about a more explanatory name?

File: index.md (hunk 3 above)
Comment (benedeki) [reply index.md:14]: Fair enough

File: _includes/page_version.html (hunk 5 above)
Comment (Zejnilovic) [reply _includes/page_version.html]: Moved back

File: _layouts/default.html [hunk 7]
```diff
@@ -1,5 +1,6 @@
 <!doctype html>
 <html class="h-100">
+  {% include page_version.html %}
```
Comment (benedeki) [inline _layouts/default.html:3]: I think you deleted this file.

Comment (Zejnilovic) [reply _layouts/default.html]: ... 🤦

repaired

**Pull request merged**